generated/Runtime/Client/Client.php

```diff
@@ -61,7 +61,8 @@ public function executeEndpoint(Endpoint $endpoint, string $fetch = self::FETCH_
                 $request = $request->withBody($body);
             } elseif (\is_resource($body)) {
                 $request = $request->withBody($this->streamFactory->createStreamFromResource($body));
-            } elseif (is_file($body)) {
+            } elseif (\strlen($body) <= 4000 && is_file($body)) {
+                // more than 4096 chars will trigger an error
```

Comment:
@damienalexandre what's the matter with 4096 chars?

Reply:
As stated in the PR description,
Calling is_file with a very long path will throw a PHP error. So this change prevent the error from happening.

Did you get an issue with this change?